waffle-compiler/src/config/inputToConfig.ts

```diff
@@ -66,7 +66,7 @@ function validate(config: any): asserts config is Config {
 const checkSourceDirectory = checkType('sourceDirectory', 'string');
 const checkOutputDirectory = checkType('outputDirectory', 'string');
 const checkNodeModulesDirectory = checkType('nodeModulesDirectory', 'string');
-const checkCompilerType = checkEnum('compilerType', ['native', 'dockerized-solc', 'solcjs']);
+const checkCompilerType = checkEnum('compilerType', ['native', 'dockerized-solc', 'solcjs', 'dockerized-vyper']);
```

Comment:
Also update in [Config.ts](https://github.com/EthWorks/Waffle/blob/733df25049036482f16f5583183ed97461b58159/waffle-compiler/src/config/config.ts#L15)